frequency-chain/provider-dashboard
# Bug/fix transferable #179

## Goal

The goal of this PR is to use correct values for transferable value for a given account.
It also updates some polkadot api packages and has some npm audit fixes.

Closes #179 

## Discussion
The transferable balance is not `accountData.free`, but rather, `accountData.free - accountData.frozen`.  See [Kusama | Account Balances: Guide](https://guide.kusama.network/docs/learn-account-balances)

## Checklist


## How To Test the Changes

1. Clone the pr branch and run the app locally:
```shell
git co bug/fix-transferable-#179
npm install
npm run dev -- --open
```
2. Connect an account that has known locked values - preferably a Provider account that has already some stake for Capacity.  If you don't have one  you can create one through the app.
3. Go to the PolkadotUI for [Frequency Paseo Accounts page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F0.rpc.testnet.amplica.io#/accounts), and look at the balances for that account.
4. Verify that the values that appear in Provider Dashboard for Transferable, Total, and Locked match what shows in the Polkadot UI.
